public/css/genericStyles.css

```diff
@@ -54,3 +54,20 @@ input[type="submit"] {
 .container {
     margin-bottom: 25px;
 }
+
+.arrowBtn{
+    border: none;
+}
+
+.arrowForm{
+    display: inline-block;
+
+}
+
+.active{
+    color:#d64405;
+}
+
+#applicantsPageHomeBtn{
```

Comment:
Should have a space between the selector and the opening {

Reply:
added spaces CSS file